templates/csharp/app-mvvm/.template.config/dotnetcli.host.json

```diff
@@ -3,9 +3,15 @@
   "symbolInfo": {
     "Framework": {
       "longName": "framework"
+    },
+    "AvaloniaVersion": {
+      "longName": "avalonia-version"
+    },
+    "MVVMToolkit": {
+      "longName": "mvvm"
     }
   },
   "usageExamples": [
-    ""
+    "--mvvm communitytoolkit"
```

Comment:
Yes, I noticed as well that it is currently not used but I think it is planned. So if you want I can remove it, but I think it would also not hurt to keep, do if it is used someday it is there